notebooks/fs.ipynb

```diff
@@ -0,0 +1,1340 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "2c013e12-a6c6-4786-aa50-900f7da77e6b",
+   "metadata": {},
+   "source": [
+    "# File System Tool\n",
+    "\n",
+    "The `uwtools` API's `fs` module provides functions to copy and link files as well as create directories. \n",
+    "\n",
+    "For more information, please see the <a href=\"https://uwtools.readthedocs.io/en/main/sections/user_guide/api/fs.html\">uwtools.api.fs</a> Read the Docs page.\n",
+    "\n",
+    "## Table of Contents\n",
+    "\n",
+    "* [Copying Files](#Copying-Files)\n",
+    "    * [Failing to copy](#Failing-to-copy)\n",
+    "    * [Using the `keys` parameter](#Using-the-keys-parameter)\n",
+    "    * [Using the `Copier` class](#Using-the-Copier-class)\n",
+    "* [Linking Files](#Linking-files)\n",
+    "    * [Failing to link](#Failing-to-link)\n",
+    "    * [Using the `keys` parameter](#Using-the-keys-parameter-)\n",
+    "    * [Using the `Linker` class](#Using-the-Linker-class)\n",
+    "* [Creating directories](#Creating-directories)\n",
+    "    * [Using the `keys` parameter](#Using-the-keys-parameter--)\n",
+    "    * [Using the `MakeDirs` class](#Using-the-MakeDirs-class)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "77864d80-e6f4-48c2-a5d5-88fc512106a9",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from uwtools.api import fs\n",
+    "from pathlib import Path"
```

Comment:
```suggestion
     "from pathlib import Path\n",
     "from shutil import rmtree\n",
     "from uwtools.api import fs\n",
    "from uwtools.api.logging import use_uwtools_logger\n",
   "\n",
    "use_uwtools_logger()"
```
This change leads to more informative output, I think, but also points out a potential issue with different output from cells if they're run more than once, which I think is something Jupyter users do often -- I know I do. This is related to the fact that this tool is idempotent, and doesn't re-do work it's already done, so if e.g. a cell copies some files and logs some nice output about what it's done, re-running the cell will produce less output since the work is already done. `rmtree()` might be useful here -- I'll make a suggestion below and see what you think.